### ISSUE TITLE
PPE Technique Video Bugs

### ISSUE THREAD

**Issue by workerbee22** [Western-Health-Covid19-Collaboration/wh_covid19_app#251]
Currently on the PPR Technique screen, you can:

- play both videos at the same time. Is this desirable? Because audio conflicts with each other.
- video once playing, can select the 'Step by Step Guide', or Method 1 or Method 2 screens and the video continues to play, with audio being heard in the background. I don't think this is desirable? Can also select the PDF screens as well, with video continuing to play in the background so audio can be heard.

Logged as the same bug, as they both relate to when video controller should be called to stop playing the video.

**Comment by lukesleeman** [comment]
Yeah, these all sounds like bugs we should fix - I'm moving this one over into ready for dev 👍

**Comment by lukesleeman** [comment]
Anybody who is looking at this should be aware that as part of #263 there is some discussion of moving over to using embeded youtube videos

**Comment by workerbee22** [comment]
I have an easy fix for this.

**Comment by workerbee22** [comment]
Additionally, once played to the end, the videos do not re-show the controls:

![image](https://user-images.githubusercontent.com/5154664/80302317-a6482900-87ec-11ea-9a1a-9b658a89df9c.png)

**Comment by workerbee22** [comment]
Moving this to milestone Version 1.1 ... as I'd recommend moving away from **chewie** as the video player to the first party plugin **video_player**. It doesn't have as much control via its controller that I'd like. It also throws lots of platform warnings on Android and iOS that are concerning.
